src/core/announcements/validation.test.ts

```diff
@@ -93,7 +93,8 @@ describe("validation", () => {
           createdAt: BigInt(+Date.now()),
           objectId: "not a valid id",
         };
-        const signedAnnouncement = await sign(announcement);
+        /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+        const signedAnnouncement = await sign(announcement as any);
```

Comment:
why do we need this `any`? can't we cast the invalid announcement to `SignedGraphChangeAnnouncement`?

Reply:
Good idea